openedx/core/djangoapps/catalog/management/commands/cache_programs.py

```diff
@@ -143,11 +146,13 @@ def get_site_program_uuids(self, client, site):  # lint-amnesty, pylint: disable
                 'status': ('active', 'retired'),
                 'uuids_only': 1,
             }
-
-            logger.info(f'Requesting program UUIDs for {site.domain}.')
-            uuids = client.programs.get(**querystring)
+            api_url = urljoin(f"{api_base_url}/", "programs")
+            logger.info('Requesting program UUIDs for %s.', site.domain)
```

Comment:
Hm, we're changing f-string in logs to the old style, may we still use the f-string instead?

Reply:
Returned the f-string syntax